docs/process/first-responder.md

```diff
@@ -5,16 +5,19 @@ We have a first responder rotation. The goals of the rotation are:
 1. Ensure community issues and PRs are answered in a timely manner.
 1. Ensure support load is shared across the whole team.
 1. Free up the rest of the team to focus on milestone work.
-1. Allocate time to less prioritized work.
+1. Give everyone a regular break from the milestone work.
 
-Each rotation is a week long. While first responder your duties are:
+Each rotation is a week long. While first responder your primary duties are:
 
 1. Triage issues.
-  * Troubleshoot or follow up on troubleshooting started by the previous first responder.
+  * Troubleshoot or follow up on troubleshooting started by the previous first responder. The current first responder should always bear responsibility for pushing troubleshooting forward, unless another team member has explicitly taken ownership.
+  * Ensure issues are labeled accurately.
   * Review the [more-information-needed](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+label%3Amore-information-needed+sort%3Aupdated-asc) and [needs-reproduction](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+sort%3Aupdated-asc+label%3Aneeds-reproduction) issues and close any that have gone 2 weeks with no new activity.
-  * See https://github.com/desktop/issue-triage/blob/master/issue-triage.md for more information on our issue triage process.
+  * See (issue-triage.md)[issue-triage.md] for more information on our issue triage process.
 1. Review community pull requests.
-1. Address technical debt, pet bugs, etc.
-1. Anything that supports the goals outlined above.
 
-It's kind of a [big block of cheese](https://www.youtube.com/watch?v=Vm9HZq53rqU) situation.
+Once those things are done, you should feel free to spend your time scratching your own itches on the project. Really wanna refactor that one monstrous component? Go for it! Wanna fix that one bug that drives you nuts? Do it! Wanna upgrade all of our dependencies? You're an awesome masochist!
```

Comment:
> Wanna upgrade all of our dependencies?

[![](https://avatars2.githubusercontent.com/in/505?s=14&u=897e2cadf0de87d7aee8407a49de73c7cd556df7&v=4) Greenkeeper](https://greenkeeper.io/) 😄

...but that’s another issue.